app.js

```diff
@@ -35,6 +35,7 @@ app.use(session({
 
 app.use(function (req, res, next) {
   var allowedOrigins = [
+    'http://46.101.83.66',
```

Comment:
likewise this seems like it should be in an external configuration.

Reply:
+1. You can have an environment variable like `PORD_DOMAIN`.